assets/_scss/core/_variables.scss

```diff
@@ -2,7 +2,7 @@
 $em-base:             10px;
 $base-font-size:      rem(17px);
 $small-font-size:     rem(14px);
-$lead-font-size:      rem(22px);
+$lead-font-size:      rem(20px);
```

Comment:
@angel yup I didn't realize we were switching to serif lead for our default. This is fine then.